docs/changelog.txt

```diff
@@ -44,8 +44,14 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - UX: List widgets now have mouse-interactive scrollbars
 - UX: You can now hold down the mouse button on a scrollbar to make it scroll multiple times.
 - UX: You can now drag the scrollbar to scroll to a specific spot
+- `spectate`: ``spectate auto-unpause`` is a new feature that will auto-dismiss pause events when enabled. This does not affect the player's ability to pause at a whim.
+- `spectate`: ``spectate auto-disengage`` is a new feature that will auto-disengage the plugin when the player interacts with the game while unpaused
+- `spectate`: ``spectate focus-jobs`` is a new feature that allows the plugin to not always follow a job, so it can trail a dwarf after they finish a job
+- `spectate`: ``spectate tick-threshold`` is a new feature that allows the player to change how long to follow a dwarf
```

Comment:
these don't need the "is a new feature that will" prefix. they're already in the "Improvements" section.

Reply:
I'll take this as meaning this is too verbose.. So I'll just change them to the same wording as the others in the same section